src/Symfony/Component/HttpKernel/DependencyInjection/LazyLoadingFragmentHandler.php

```diff
@@ -42,23 +42,24 @@ public function __construct(ContainerInterface $container, RequestStack $request
     /**
      * Adds a service as a fragment renderer.
      *
+     * @deprecated since version 3.3, to be removed in 4.0
+     *
      * @param string $name     The service name
      * @param string $renderer The render service id
      */
     public function addRendererService($name, $renderer)
     {
-        $this->rendererIds[$name] = $renderer;
```

Comment:
removing the implementation is wrong. It means that the feature does not work anymore, which is a BC break